### DISCUSSION TITLE
Best approach to running Cicero by cluster

### DISCUSSION THREAD

**Discussion by wudustan** [stuart-lab/signac#949] [Q&A]
Hi,

I am using Signac to define clusters in my scATAC data. I would like to run Cicero on each cluster and compare the CCANs between them.

At what point in the [Cicero Signac vignette](https://satijalab.org/signac/articles/cicero.html) does it make sense to do this? Do I need to split the Signac object into individual objects by cluster, or should it do it downstream after I convert to Cicero CDS using `make_cicero_cds()`.

Thank you

I am also receiving the following warnings, is this OK: 

```
Warning: Monocle 3 trajectories require cluster partitions, which Seurat does not calculate. Please run 'cluster_cells' on your cell_data_set object
Warning messages:
1: In .check_reddim_names(x, value, withDimnames) :
  non-NULL 'rownames(value)' should be the same as 'colnames(x)' for 'reducedDim<-'. This will be an error in the next release of Bioconductor.
2: In .check_reddim_names(x, value[[v]], withDimnames = TRUE, vname = sprintf("value[[%s]]",  :
  non-NULL 'rownames(value[[1]])' should be the same as 'colnames(x)' for 'reducedDims<-'. This will be an error in the next release of Bioconductor.
```

**Comment by timoast** [comment]
Hi, I think you should ask the Cicero and Monocle developers this question

**Reply by wudustan** [reply]
Hi Tim,

I agree. I did post this question on the cicero-users Google group but I've yet to receive a reply. I know that while Hannah does try and answer each query, she doesn't get the chance to do so very often and I suppose I'm wondering if you have an insight/experience with running these kinds of analyses while at the same time appreciating that you don't develop or maintain Cicero.

I appreciate your help with my many Signac queries and I definitely owe you multiple beers at this point!

Yura

**Reply by timoast** [reply]
Ok in that case, my opinion is:

Cicero is a correlation-based method, which depends on having heterogeneity between cells to find the co-accessible networks. If you subset to a more homogeneous cluster of cells, you may not be able to detect a correlation between peak accessibility. I would suggest identifying CCANs using all cells, then quantifying the total accessibility of each network in the different clusters and comparing that.
